var/spack/repos/builtin/packages/wx/package.py

```diff
@@ -41,18 +42,24 @@ class Wx(Package):
     version('3.0.2', '6461eab4428c0a8b9e41781b8787510484dea800')
     version('3.0.1', '73e58521d6871c9f4d1e7974c6e3a81629fddcf8')
 
+    version('develop', git='https://github.com/wxWidgets/wxWidgets.git', branch='master')
+
     depends_on('gtkplus')
 
-    def make_wx(self):
-        make()
+    parallel = False
```

Comment:
No, I just don't think it's possible to keep the old logic with AutotoolsPackage